src/status_im/ui/screens/chat/views.cljs

```diff
@@ -32,20 +32,16 @@
 
 (defn topbar [current-chat]
   [topbar/topbar
-   {:content      [toolbar-content/toolbar-content-view]
-    :show-border? true
-    :initial-title-padding 56
-    :navigation   {:icon                :main-icons/arrow-left
-                   :accessibility-label :back-button
-                   :handler
-                   #(re-frame/dispatch [:navigate-to :home])}
-    :accessories  [{:icon                :main-icons/more
-                    :accessibility-label :chat-menu-button
-                    :handler
-                    #(re-frame/dispatch [:bottom-sheet/show-sheet
-                                         {:content (fn []
-                                                     [sheets/actions current-chat])
-                                          :height  256}])}]}])
+   {:content           [toolbar-content/toolbar-content-view]
+    :border-bottom     true
```

Comment:
do we have any statistics or info from @errorists what is default , with border or without ?

Reply:
without border is mostly only used in Onboarding, you know the kind of views where we didn't expect the content to scroll. Having the border on is a sensible default 👍

Reply:
it's mostly like so, 
Regular Top Bar:
• with border (default)
• without border
Extended Top Bar
• with the minimised state / shadow on scroll

Reply:
thanks, so it seems its better to have a border by default, and have a hide-border property